cartography/intel/aws/rds.py

```diff
@@ -98,6 +105,132 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
             AWS_ACCOUNT_ID=current_aws_account_id,
             aws_update_tag=aws_update_tag
         )
+        _attach_ec2_security_groups(neo4j_session, rds, aws_update_tag)
+        _attach_ec2_subnet_groups(neo4j_session, rds, aws_update_tag)
+    _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
+
+
+def _attach_ec2_subnet_groups(neo4j_session, instance, aws_update_tag):
+    """
+    Attach RDS instance to its EC2 subnets
+    """
+    attach_rds_to_subnet_group = """
+    MERGE(sng:DBSubnetGroup{id:{DBSubnetGroupName}})
```

Comment:
Are the SubnetGroup an RDS specific concept? If so, should we drive toward consistency and rename as RDSSubnectGroup ?

Reply:
I believe so.  I don't want to name it an RDSSubnetGroup though because the [aws datatype](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBSubnetGroup.html) is actually called DBSubnetGroup.

Reply:
My thinking there is to be more explicit about what nodes are and coming from. We don't know what the future will bring and also what users will add. I do admit that we may not have considered that earlier on but something to revisit as we move forward. Looping @evandavis so we can reach consensus